client/src/client_sync/v17/mod.rs

```diff
@@ -245,3 +246,40 @@ pub enum SetBanCommand {
     Add,
     Remove,
 }
+
+/// Args for the `importmulti` method
+#[derive(Clone, Debug, Default, Deserialize, PartialEq, Serialize)]
+pub struct ImportMultiRequest {
```

Comment:
A question for @tcharding what is the reason for these argument structs being in `client` and not in `types`?

Reply:
Because they are used with the `client` and if one writes their own `client` (which I hope folk will) and just depend on `types` we don't want a bunch of 'random' stuff in there. On the other hand wsince we have the `client-sync` feature we could expose them and then writers of a `client` could still depend on our client to grab them? I'd be open to that. (By expose I mean put them in `lib.rs` or someplace so they are not hidden by the feature.)